pages/common/sqsc.md

```diff
@@ -0,0 +1,31 @@
+# sqsc
+
+> A command line AWS SQS client.
```

Comment:
What does `SQS` stand for?

Reply:
Simple Queue Service, basically hosted ActiveMQ.

Reply:
Ah, ok! Perhaps we could update the description then?